DDMockiOS.podspec

```
@@ -29,7 +29,7 @@ Pod::Spec.new do |spec|
 
   spec.ios.deployment_target = '11.0'
 
-  spec.source_files = 'DDMockiOS'
+  spec.source_files = 'Sources/DDMockiOS/**/*'
```

I think this needs to be:
```suggestion
  spec.source_files = 'Sources/DDMockiOS'
```
or:
```suggestion
  spec.source_files = 'Sources/DDMockiOS/**/*.swift'
```
The Info.plist file is getting identified as source incorrectly and causing a build error